################################################################################
SHORT DESCRIPTION: 
################################################################################
Don't force XEN_ROOT path to be absolute.

################################################################################
LONG DESCRIPTION: 
################################################################################
This is needed in our OE build environment.

################################################################################
CHANGELOG 
################################################################################

################################################################################
REMOVAL 
################################################################################

################################################################################
UPSTREAM PLAN
################################################################################
No.

################################################################################
INTERNAL DEPENDENCIES 
################################################################################

################################################################################
PATCHES 
################################################################################
Index: xen-4.10.0/Config.mk
===================================================================
--- xen-4.10.0.orig/Config.mk
+++ xen-4.10.0/Config.mk
@@ -1,8 +1,8 @@
 # -*- mode: Makefile; -*-
 
-ifeq ($(filter /%,$(XEN_ROOT)),)
-$(error XEN_ROOT must be absolute)
-endif
+#ifeq ($(filter /%,$(XEN_ROOT)),)
+#$(error XEN_ROOT must be absolute)
+#endif
 
 # Convenient variables
 comma   := ,
